perf/specjbb/run/options/specjbb-multijvm-simple.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+# This will configure a basic run of SPECjbb in multi-jvm mode
+# Therefore, the topography of the run includes 1 group consisting of
+# 1 TransactionInjector and 1 Backend
+
+export GROUP_COUNT=1
```

Comment:
Worth commenting 'why' these parameters have been set?

*Comment applies to all parameters

Reply:
I included a comment on lines 2-4 explaining some of the parameters. However, I see that params on lines 14-16 need more detail since they imply a hardware config  requirement for a successful run. I'll also surface that hardware requirement to the playlist.xml file so that it is explicit.